addons/l10n_uy/tests/test_doc_types.py

```diff
@@ -0,0 +1,90 @@
+from odoo import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("-at_install", "post_install", "post_install_l10n")
+class TestDocTypes(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='uy'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.partner_test = cls.env["res.partner"].create({
+            "name": "IEB Internacional",
+            "l10n_latam_identification_type_id": cls.env.ref("l10n_uy.it_rut").id,
+            "vat": "218435730016",
+            "street": "Bach 0",
+            "city": "Aeroparque",
+            "state_id": cls.env.ref("base.state_uy_02").id,
+            "country_id": cls.env.ref("base.uy").id,
+            "email": "rut@example.com",
+        })
+
+        cls.service_vat_22 = cls.env["product.product"].create({
+            "name": "Virtual Home Staging (VAT 22)",
+            "list_price": 38.25,
+            "standard_price": 45.5,
+            "type": "service",
+            "default_code": "VAT 22",
+        })
+
+        cls.invoice = cls._create_invoice(
```

Comment:
i think it would be better to just create an account.move directly instead of using a function for that

```suggestion
        cls.invoice = invoice = self.env['account.move'].create({
            'partner_id': ...,
            'move_type': ...,
            'ref': 'reference',
            'invoice_line_ids': [
                Command.create({
                    'quantity': 1,
                    'price_unit': ....,
                    'name': 'Line 1',
                }),
          ]
```
(the indent is a bit broken but you see the point ahah)